### PR TITLE
Anonymizing

### PR DESCRIPTION
**Pull request by zhiyil1230**
1, removed notebooks related stuff and pdf 
2, apply anonymizer with Graphcore as keyword

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+An optimised hybrid GNN/Transformer model for molecular property prediction using XXXX-1 IPUs, trained on the [PCQM4Mv2](https://arxiv.org/abs/2103.09430) dataset. The flexible hybrid model closely follows the [General, Powerful, Scalable (GPS) framework](https://arxiv.org/abs/2205.12454) and combines the benefits of both message passing and attention layers for graph-structured input data training.
 
 ## Running the model [![Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/3CGjC5E)
 
 ### Setup
 
-This model is optimised for Graphcore IPUs and requires the Graphcore's Poplar SDK to run. You can access IPUs through [Paperspace](https://www.paperspace.com/graphcore), using the button above, or [G-Core](https://gcore.com/partners/graphcore).
+This model is optimised for XXXX-1 IPUs and requires the XXXX-1's Poplar SDK to run.
```
Comment (zhiyil1230) [inline README.md]: I think we could actually mention Graphcore in this setup stage?

Comment (s-maddrellmander) [reply README.md]: Yeah put Graphcore back in here.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 Create a virtual environment and install the Poplar SDK, including the TensorFlow 2 wheels from inside the SDK directory:
 
 ```shell
 virtualenv --python python3.6 .gps_venv
 source .gps_venv/bin/activate
-source <path to the Graphcore SDK>/enable
+source <path to the XXXX-1 SDK>/enable
```
Comment (s-maddrellmander) [inline README.md]: And admit back here too I think.

File: argparser.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 Graphcore Ltd. All rights reserved.
+# Copyright (c) 2022 XXXX-1 Ltd. All rights reserved.
 
 from email.policy import default
```
Comment (s-maddrellmander) [inline argparser.py]: Coincidently I think we probably don't want this import? What's going on here? This can't be used?

**Review by s-maddrellmander — COMMENTED** [review]
Just those last few in the README, otherwise it's all good!
I've tested the scripted it downloads the data, and running on pre-processed data it complies and runs absolutely fine.

File: argparser.py (hunk 3 above)
Comment (zhiyil1230) [reply argparser.py]: Yeah looks like it was not used at all, not sure where it comes from. And also the
`from random import choices` was not used.
In the parse.add_argument part we do have the default and choices options.

File: README.md [hunk 4]
```diff
@@ -2,11 +2,11 @@
 
 An optimised hybrid GNN/Transformer model for molecular property prediction using Graphcore IPUs, trained on the [PCQM4Mv2](https://arxiv.org/abs/2103.09430) dataset. The flexible hybrid model closely follows the [General, Powerful, Scalable (GPS) framework](https://arxiv.org/abs/2205.12454) and combines the benefits of both message passing and attention layers for graph-structured input data training.
 
-## Running the model [![Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/3CGjC5E)
+## Running the model
 
 ### Setup
 
-This model is optimised for Graphcore IPUs and requires the Graphcore's Poplar SDK to run. You can access IPUs through [Paperspace](https://www.paperspace.com/graphcore), using the button above, or [G-Core](https://gcore.com/partners/graphcore).
+This model is optimised for Graphcore IPUs (available on [Paperspace](https://www.paperspace.com/graphcore)) and requires the Graphcore's Poplar SDK to run.
```
Comment (s-maddrellmander) [inline README.md:9]:
```suggestion
This model is optimised for Graphcore IPUs (available on [Paperspace](https://www.paperspace.com/graphcore)) and requires the Graphcore's Poplar SDK to run.
Paperspace has instances with Graphcore IPUs available to use, free of charge, for up to 6 hours. 
```

**Pull request closed without merging**